deploy/helm/sumologic/conf/setup/setup.sh

```diff
@@ -1,5 +1,18 @@
 #!/bin/bash
 
+readonly DEBUG_MODE=${DEBUG_MODE:="false"}
+readonly DEBUG_MODE_ENABLED_FLAG="true"
+
+if [[ ${DEBUG_MODE,,} == ${DEBUG_MODE_ENABLED_FLAG} ]]; then
```

Comment:
`${DEBUG_MODE,,}`  ->   `${DEBUG_MODE}`

Reply:
That's on purpose, see the other comment.

Reply:
Let me add a comment in the code.